src/xcode/ENA/ENA/Resources/Localization/en.lproj/Localizable.strings

```diff
@@ -1793,7 +1793,7 @@
 
 "Tabbar_Diary_Title" = "Journal";
 
-"Tabbar_CheckIn_Title" = "Check In";
+"Tabbar_CheckIn_Title" = "Check-in";
```

Comment:
Not sure if we are allowed to change translated texts.

Reply:
@janetback Can you answer that?